test/spec/tagspec.js

```diff
@@ -86,6 +86,14 @@ describe("Cloudinary.ImageTag", function() {
     tag = new cloudinary.ImageTag('image_id', config).toHtml();
     return expect(tag).toBe(`<img src="${DEFAULT_UPLOAD_PATH}image_id">`);
   });
+  it("should escape quotes in html attributes", function() {
+    var tag;
+    tag = new cloudinary.ImageTag('image_id', {
+      'cloud_name': 'test123',
```

Comment:
Use the `config` variable.

Reply:
Used it with Object.assign to add "alt" param